src/Bridges/Constraint/bridges/square.jl

```diff
@@ -356,3 +413,79 @@ function MOI.get(
     end
     return dual
 end
+
+function MOI.set(
+    model::MOI.ModelLike,
+    attr::MOI.ConstraintDualStart,
+    bridge::SquareBridge,
+    value,
+)
+    # Our output will be a dense square matrix.
+    dim = MOI.side_dimension(bridge.square_set)
+    offset = length(_square_offset(bridge.square_set))
+    @assert length(value) == offset + dim^2
+    dual = Vector{eltype(value)}(undef, offset + div(dim * (dim + 1), 2))
+    # Start by converting the triangular dual to the square dual, assuming that
+    # all elements are symmetrical.
+    for i in 1:offset
+        dual[i] = value[i]
+    end
+    k = offset
+    sym_index = 1
+    for j in 1:dim, i in 1:j
+        k += 1
+        upper_index = offset + i + (j - 1) * dim
+        lower_index = offset + j + (i - 1) * dim
+        if i == j
+            dual[k] = value[upper_index]
+        elseif sym_index <= length(bridge.sym) &&
+               bridge.sym[sym_index].first == (i, j)
+            # The PSD constraint uses only the upper triangular part. Therefore,
+            # for KKT to hold for the user model, the dual given by the user
+            # needs to be attributed to the upper triangular entry. For example,
+            # suppose the constraint is
+            #   [0 x; y 0] in PositiveSemidefiniteConeSquare(2).
+            # If the dual is
+            #   [λ1 λ3; λ2 λ4]
+            # then we have `y λ2 + x λ3` in the Lagrangian.
+            #
+            # In the bridged model, the constraint is
+            #   [0, x, 0] in PositiveSemidefiniteConeTriangle(2).
+            #   [x - y] in Zeros(1)
+            # If the dual is
```

Comment:
I don't think you need to copy-past the comment, just say that we are doing the inverse of what in the getter